test/e2e/ingress.go

```diff
@@ -159,7 +160,13 @@ var _ = framework.KubeDescribe("Loadbalancing: L7 [Feature:Ingress]", func() {
 			// but we want to allow easy testing where a user might've hand
 			// configured firewalls.
 			if framework.ProviderIs("gce", "gke") {
-				ExpectNoError(gcloudCreate("firewall-rules", fmt.Sprintf("ingress-80-443-%v", ns), framework.TestContext.CloudConfig.ProjectID, "--allow", "tcp:80,tcp:443", "--network", "e2e"))
+				// TODO: The KUBE_GCE_NETWORK env var is exported for both
+				// gce/gke e2e test runs, pipe it through just like ProjectID.
+				network := os.Getenv("KUBE_GCE_NETWORK")
```

Comment:
adding a flag is like 3 more lines of code: https://github.com/kubernetes/kubernetes/commit/fb56095683787744f6a2b86996fac0216d50a65f

I'd prefer to just do it correctly.